store.go

```diff
@@ -2929,7 +2933,11 @@ func (s *store) Layers() ([]Layer, error) {
 
 	for _, s := range lstores {
 		store := s
-		store.RLock()
+		if store == lstore {
```

Comment:
I don't think this comparison is ever true, so I don't think this hunk changes the behavior.

Reply:
Ok , that's right, I will upate this comparison.